pkg/gitreceive/k8s_util.go

```diff
@@ -222,3 +226,21 @@ func waitForPodCondition(c *client.Client, ns, podName string, condition func(po
 		return false, nil
 	})
 }
+
+func progress(msg string) chan bool {
+	tick := time.Tick(8000 * time.Millisecond)
+	quit := make(chan bool)
+	go func() {
+		for {
+			select {
+			case <-quit:
+				close(quit)
+				return
+			case <-tick:
+				fmt.Println(msg)
```

Comment:
ok, then can you print something like a carriage return or a `.` character instead?